api/app/signals/apps/signals/models/category_departments.py

```diff
@@ -0,0 +1,20 @@
+from django.contrib.gis.db import models
+from django.core.exceptions import ValidationError
+
+
+class CategoryDepartment(models.Model):
+    category = models.ForeignKey('signals.Category', on_delete=models.CASCADE)
+    department = models.ForeignKey('signals.Department', on_delete=models.CASCADE)
+
+    is_responsible = models.BooleanField(default=False)
+    can_view = models.BooleanField(default=False)
```

Comment:
can_view voegt op dit moment niets toe, want deze is altijd True. (gezien de validatie in save()).

Reply:
Ik heb de validatie in de save voor nu weggehaald. Zo kan de koppeling wel blijven bestaan maar is het niet meer mogelijk om items in deze categorie te bekijken.

Wil de `can_view` wel behouden aangezien deze, net zoals `is_responsible`, duidelijk aangeeft wat de bedoeling is. In vervolg tickets worden deze flags bewerkbaar via de API.